pkg/externalresource/external_resource_test.go

```diff
@@ -95,12 +110,15 @@ func TestExternalResource_LoadMarkdownContent(t *testing.T) {
 				assert.Error(t, tt.externalResourceFSM.LoadMarkdownContent(manifestPath))
 			} else {
 				assert.NoError(t, tt.externalResourceFSM.LoadMarkdownContent(manifestPath))
-				sDec, err := b64.StdEncoding.DecodeString(tt.externalResourceFSM.ExternalResource.Notes.Markdown)
-				if err != nil {
-					t.Fatal(err)
+				if tt.externalResourceFSM.ExternalResource.Notes != nil {
```

Comment:
I am not sure if it is appropriate to do this check or to directly compare the expected object vs the one we get

Reply:
If the function reads and set the value, I'd assert that the complete object is in the state that we want